test/connectors/FinancialInstitutionsConnectorSpec.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import helpers.WireMockServerHandler
+import play.api.Application
+import play.api.http.Status.OK
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class FinancialInstitutionsConnectorSpec extends SpecBase with WireMockServerHandler {
+
+  lazy val app: Application = applicationBuilder()
+    .configure(
+      conf = "microservice.services.crs-fatca-fi-management.port" -> server.port(),
+      "microservice.services.crs-fatca-fi-management.bearer-token" -> "local-token",
+      "auditing.enabled"                                           -> "false"
+    )
+    .build()
+
+  lazy val connector: FinancialInstitutionsConnector =
+    app.injector.instanceOf[FinancialInstitutionsConnector]
+
+  "FinancialInstitutionsConnector" - {
+
+    "viewFinancialInstitutions" - {
+
+      "must return status as OK for viewFIs" in {
+        val subscriptionId = "XE512345678"
+        stubResponse(
+          s"/financial-institutions/$subscriptionId",
+          OK
+        )
+        val result = connector.viewFis(subscriptionId)
+        result.futureValue.status mustBe OK
```

Comment:
There isn't much here but we testing simply the stub, which is fine but mainly we need the `FinancialInstitutionsServiceSpec`

Reply:
I did think this was a bit light - i'll put something in, without a frontend use it seemed odd

Reply:
added - had a small bit of faff because i had one of the models wrong so the validation was throwing errors aha